llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/VectorExt/IR/VectorExtAttrs.td

```diff
@@ -293,6 +295,13 @@ def NestedLayoutAttr : IREEVectorExt_Attr<"NestedLayout",
     `>`
   }];
 
+  let extraClassDeclaration = [{
+    // Returns the subgroup/lane ids delinearized from a single linearized
+    // thread ID.
+    FailureOr<ValueRange>
+    computeThreadIds(Value threadId, RewriterBase &rewriter) const;
```

Comment:
When does this fail? Can we have this in the documentation?

Reply:
Can't fail anymore, forgot to drop after moving it from the interface.